src/theme/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const DEFAULT_FONT_FAMILY = 'Quicksand';
+export const DEFAULT_FONT_SIZE = '0.875rem';
+export const DEFAULT_FONT_WIGHT = 'bold';
```

Comment:
typo

Reply:
@damianlewandowski well we can do this way but I think there will be cases where we will need to override

Reply:
I agree, I don't want to completely remove overrides, just give a way of using those dark-grey and green colors, which are main and secondary colors by design, freely.

For example when we have color "green" used both in button and in header. Let's not define two overrides for "button" AND for "header", let's instead have this color easily accessible from `theme.palette` and use it in JSS.